ecommerceproj/theretailerapp/templates/theretailerapp/customer_list.html

```diff
@@ -1,40 +1,51 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  {% block title %}<title>The retailer app</title>{% endblock %}
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <!-- Add additional CSS in static file -->
-  {% load static %}
-  <link rel="stylesheet" href="{% static 'css/styles.css' %}">
-</head>
-<body>
-<div class = "container">
-<a class='button button2' href="product" >Product List</a>
-<h1>Customer List</h1>
-<div>
-{% block content %}
-  {% if customer_list %}
-  <ul>
-    {% for customer in customer_list %}
-      <li>
-        <p>Name: {{ customer.first_name }}&nbsp;{{customer.last_name}}</p>
-        <p>Shipping address: {{customer.shipping_address}}</p>
-        <p>Billing address: {{customer.billing_address}}</p>
-        <p>Phone_number: {{customer.ph_number}}</p>
-        <p>Email: {{customer.email}}</p>
-        <p>Country: {{customer.counrty}}</p>
-        <p>Created at: {{customer.created_at}}</p>
-        <p>Updated at: {{customer.updated_at}}</p>
-      </li>
-    {% endfor %}
-  </ul>
-  {% else %}
-    <p>There are no customers.</p>
-  {% endif %}
-{% endblock %}
-  </ul>
-</div>
-</div>
-</body>
+  <head>
+    {% block title %}
+    <title>The retailer app</title>
+    {% endblock %}
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <!-- Add additional CSS in static file -->
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'css/styles.css' %}" />
+  </head>
+  <body>
+    <div class="container">
+      <a class="button button2" href="product">Product List</a>
+      <h1>Customer List</h1>
+      <div>
+        {% block content %} {% if customer_list %}
```

Comment:
Please move `{% if customer_list %}` to next line and add proper indentation afterwards.